config.fnl

```diff
@@ -0,0 +1,455 @@
+(local windows (require :windows))
+(local {:concat concat
+        :logf logf} (require :lib.functional))
+
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;; Default Config
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;
+;; - It is not recommended to edit this file.
+;; - Changes may conflict with upstream updates.
+;; - Create a ~/.hammerspoon/private/config.fnl file instead.
+;;
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+
+
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;; Table of Contents
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+
+;; [x] w - windows
+;; [x] |-- w - Last window
+;; [x] |-- cmd + hjkl - jumping
+;; [x] |-- hjkl - halves
+;; [x] |-- alt + hjkl - increments
+;; [x] |-- shift + hjkl - resize
+;; [x] |-- n, p - next, previous screen
+;; [x] |-- shift + n, p - up, down screen
+;; [x] |-- g - grid
+;; [x] |-- m - maximize
+;; [x] |-- c - center
+;; [x] |-- u - undo
+;;
+;; [x] a - apps
+;; [x] |-- e - emacs
+;; [x] |-- g - chrome
+;; [x] |-- f - firefox
+;; [x] |-- i - iTerm
+;; [x] |-- s - Slack
+;; [x] |-- b - Brave
+;;
+;; [x] j - jump
+;;
+;; [x] m - media
+;; [x] |-- h - previous track
+;; [x] |-- l - next track
+;; [x] |-- k - volume up
+;; [x] |-- j - volume down
+;; [x] |-- s - play\pause
+;; [x] |-- a - launch player
+;;
+;; [x] x - emacs
+;; [x] |-- c - capture
+;; [x] |-- z - note
+;; [x] |-- f - fullscreen
+;; [x] |-- v - split
+;;
+;; [x] cmd-n - next-app
+;; [x] cmd-p - prev-app
+
+
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;; Actions
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+
+(fn activator
+  [app-name]
+  (fn activate []
+    (windows.activate-app app-name)))
+
+
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;; General
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+
+(local music-app
+       "Spotify")
```

Comment:
Link to the docs with the private/config.fnl example showing how to override default music app

Reply:
As of https://github.com/agzam/spacehammer/pull/35 we are creating a ~/.spacehammer/config.fnl automatically so users can change it right there.